src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -721,6 +722,41 @@ var adyenHelperObj = {
       isSuccessful: false,
     };
   },
+
+  executeCall(serviceType, requestObject) {
+    const service = this.getService(serviceType);
+    if (service === null) {
+      throw new Error(`Could not create ${serviceType} service object`)
+    }
+    const maxRetries = constants.MAX_API_RETRIES;
+    const apiKey = AdyenConfigs.getAdyenApiKey();
+    const uuid = UUIDUtils.createUUID();
+    service.addHeader('Content-type', 'application/json');
+    service.addHeader('charset', 'UTF-8');
+    service.addHeader('X-API-KEY', apiKey);
+    service.addHeader('Idempotency-Key', uuid);
+
+    let callResult;
+    // retry the call until we reach max retries OR the callresult is OK
+    for(let nrRetries=0; nrRetries<maxRetries && !callResult?.isOk(); nrRetries++) {
```

Comment:
Could you please add spaces here? for a second I thought `nrRetries<maxRetries` was like generics in Java :P